lib/airbrake-ruby/notifier.rb

```diff
@@ -30,7 +30,7 @@ class Notifier
     def initialize(user_config)
       @config = (user_config.is_a?(Config) ? user_config : Config.new(user_config))
 
-      unless [@config.project_id, @config.project_key].all?
```

Comment:
Try a newer version (1.4.2). It was fixed by https://github.com/airbrake/airbrake-ruby/pull/89 :)
Sorry for the inconvenience.